src/tokenize/tokenize.ts

```diff
@@ -226,7 +226,7 @@ function* _tokenize(readline: () => string, encoding?: string, filename = "<toke
         // hence `line` itself will always be overwritten at the end
         // of this loop.
         lasline = line;
-        line = readline() || ""; // readline will return undefined when it's done
+        line = readline.next().value || ""
```

Comment:
I think this is a good change internally. If when we expose this to python we'll need to wrap the tokenize api to match python. 

At the bottom of this file there's a `readline` function we should change and move it to a better place.